src/Components/Patient/PatientConsentRecords.tsx

```diff
@@ -37,12 +36,20 @@ export default function PatientConsentRecords(props: {
 
   const fileUpload = useFileUpload({
     type: "CONSENT_RECORD",
+    allowedExtensions: ["pdf", "jpg", "jpeg", "png"],
   });
 
   const fileManager = useFileManager({
     type: "CONSENT_RECORD",
     onArchive: async () => {
       refetch();
+      refetchFiles();
```

Comment:
Explore the possibility of Joining Applicable Files with the Consent Records on the backend